.github/workflows/changelog_verifier.yml

```diff
@@ -0,0 +1,18 @@
+name: "Changelog Verifier"
+on:
+  pull_request:
+    types: [opened, edited, review_requested, synchronize, reopened, ready_for_review, labeled, unlabeled]
```

Comment:
nit: do we actually need all these types? I think we can keep like this for now but referencing to what the OSD core has, `opened, edited, reopened` should be good. But yes, lets keep as it for now to see if this works.

- https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.github/workflows/opensearch_changelog_workflow.yml#L5

Reply:
I yanked this from the security repo so we should also update the list there as well: https://github.com/opensearch-project/security/blob/main/.github/workflows/changelog_verifier.yml#L4

Reply:
Changed the list here to `[opened, edited, reopened]`